tests/e2e.bats

```diff
@@ -1,5 +1,24 @@
 #!/usr/bin/env bats
 
+# Define global variables (for use in teardown)
+setup() {
```

Comment:
As pointed out by aojea, I don't think we need the setup and teardown functions for writing this test. All of this behaviour should ideally be part of the test function itself.